soccer/src/soccer/planning/params.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <rj_param_utils/param.hpp>
+
+namespace trajectory_utils {
+    constexpr auto kMaxIterations = "trajectory_utils";
+    constexpr auto kExpectedDt = "trajectory_utils";
```

Comment:
These variables should be deleted. There should be a constant for `kPlanningParams` (the string constant is the module name).